src/Http/Middleware/EnsureFrontendRequestsAreStateful.php

```diff
@@ -52,6 +52,7 @@ protected function frontendMiddleware()
             \Illuminate\Session\Middleware\StartSession::class,
             config('sanctum.middleware.validate_csrf_token'),
             config('sanctum.middleware.verify_csrf_token', \Illuminate\Foundation\Http\Middleware\VerifyCsrfToken::class),
+            config('sanctum.middleware.ensure_not_logged_out', EnsureDeviceHasNotBeenLoggedOut::class),
```

Comment:
I feel it might be best to use the same conversion as Laravel Jetstream:

https://github.com/laravel/jetstream/blob/a57287944cd0168cd6e8f9237ef4965711748515/routes/inertia.php#L27-L29